src/portal/src/i18n/lang/tr-tr-lang.json

```diff
@@ -1233,15 +1233,15 @@
         "ACTION": "ACTION"
     },
     "IMMUTABLE_TAG": {
-        "IMMUTABLE_RULES": "Immutable rules",
+        "IMMUTABLE_RULES": "Immutable tag rules",
         "ADD_RULE": "ADD RULE",
         "ADD_RULE_HELP_1": "Click the ADD RULE button to add a rule.",
         "EDIT": "Edit",
         "DISABLE": "Disable",
         "ENABLE": "Enable",
         "DELETE": "Delete",
-        "ADD_TITLE": "Add Immutable Rule",
-        "ADD_SUBTITLE": "Specify a immutable rule for this project. All immutable rules are independently calculated and each rule can be applied to a selected list of repositories.",
+        "ADD_TITLE": "Add Immutable Tag Rule",
+        "ADD_SUBTITLE": "Specify an immutable tag rule for this project. All immutable tag rules are independently calculated and each rule can be applied to a selected list of repositories.",
```

Comment:
what does " All immutable tag rules are independently calculated " mean?

Reply:
I have communicated with Pm and owner, and I am thinking about it together.